### PR TITLE
Adds test coverage topic

### PR DESCRIPTION
**Pull request by pedpess**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -450,6 +451,27 @@ assertThat(nextStartedServiceIntent)
     ... so on
 ```
 
+### Test coverage
+
+Why not to have a big picture of how your tests are manipulating your code? Having a code coverage tool might help you to avoid to not have a bad time in the future with your Android project. In order to generate some nice reports from the tests we use **[JaCoCo](http://www.eclemma.org/jacoco/)**. Basically, JaCoCo stands for Java Code Coverage, but it doesn't mean that you can't test your Kotlin projects as well. It runs your tests written using **Roboeletric and Espresso**. There are few other tools for test coverage available, one of them is [SonarQube](https://www.sonarqube.org/), deserves a check.
```
Comment (tomaszpolanski) [inline README.md]: Regarding the par `Roboeletric and Espresso`. Didn't we encourage not to use Roboeletric in the other PR for Best Practices?

**Comment by pedpess** [comment]
Should we move this test coverage tip forward?

File: README.md (hunk 1 above)
Comment (pedpess) [reply README.md]: Gonna fix this. But do you still think is a relevant topic to add knowing that there is no test coverage tool for Kotlin still?

Comment (peter-tackage) [reply README.md]: Code coverage is still relevant, once there's something available for Kotlin and we think it's worthwhile using, then we can add it.

**Comment by partounian** [comment]
Is there a test coverage tool for Kotlin now?

**Pull request closed without merging**